content/docs/command-reference/stage/list.md

```diff
@@ -67,9 +66,9 @@ DVC also uses this command to provide autocompletion in `bash` and `zsh`.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-## Example: List stages
+## Examples:
 
-Listing stages from the default `./dvc.yaml` file from the repo.
```

Comment:
`Default` is a better term to my mind. Again, what is the reason/motivation for this PR. may be I'm missing the context

Reply:
OK, but there can only be one `./dvc.yaml` file, so it doesn't make sense to put "default" before that. Changed to

> The default behavior is to list stages from `./dvc.yaml`